deps/samba.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "samba",
+    "buildsystem": "simple",
+    "build-commands": [
+        "./buildtools/bin/waf configure --prefix=/app --disable-python --without-ad-dc --without-acl-support --without-ldap --without-ads --without-pam --without-systemd",
+        "./buildtools/bin/waf build -j $FLATPAK_BUILDER_N_JOBS",
+        "./buildtools/bin/waf install"
+    ],
+    "cleanup": [
+        "/bin",
+        "/lib/python2.7",
+        "/libexec",
+        "/share",
+        "/include",
+        "/lib/*.so"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://download.samba.org/pub/samba/stable/samba-4.7.4.tar.gz",
```

Comment:
Could you update samba? The latest release from 4.7.x branch is 4.7.11. The latest overall release is 4.9.2.